### PR TITLE
feat: Adding Secrets Manager module under Security

### PR DESCRIPTION
**Pull request by rodrigobersa**
#### Which issue(s) this PR fixes:

Fixes #

#### Quality checks

- [x] My content adheres to the style guidelines
- [ ] I ran `make test` or `make e2e-test` and it was successful

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: website/docs/security/secrets-management/secrets-manager/create-secret.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: "Storing secrets in AWS Secrets Manager"
+sidebar_position: 61
+---
+
+To store a secret in AWS Secrets Manager, you can either use the AWS Console or the AWS CLI. We will cover both scenarios below.
+
+### AWS Console
```
Comment (niallthomson) [inline website/docs/security/secrets-management/secrets-manager/create-secret.md:8]: Maintaining console screenshots is more overhead, lets just remove this section and stick to the CLI

File: website/docs/security/secrets-management/secrets-manager/create-secret.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+
+Leave the rest of the configuration with the default values, then click **Create**.
+
+### AWS CLI
+
+If you prefere to use the AWS CLI, run the command like the example below in you Cloud9 Environment.
+
+```bash
+$ aws secretsmanager create-secret --name "eks-workshop/catalog-secret" --secret-string '{"username":"catalog_user", "password":"default_password"}' --region $AWS_REGION
```
Comment (niallthomson) [inline website/docs/security/secrets-management/secrets-manager/create-secret.md:31]: Can you create the secret with a name of something like `$EKS_CLUSTER_NAME/catalog-secret` instead so we don't get conflicts in CI?

File: website/docs/security/secrets-management/secrets-manager/ascp.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
+---
+title: "AWS Secrets and Configuration Provider (ASCP)"
+sidebar_position: 62
+---
+
+If you ran the `prepare-environment` script detailed in a [previous step](./index.md), it has already installed the AWS Secrets and Configuration Provider (ASCP) for the Kubernetes Secrets Store CSI Driver that's required for this lab.
```
Comment (niallthomson) [inline website/docs/security/secrets-management/secrets-manager/ascp.md:6]: "If you ran" is not an if, reword to something like "When you ran"

File: website/docs/security/secrets-management/secrets-manager/ascp.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+NAME                                                   DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR            AGE
+daemonset.apps/secrets-store-csi-driver-provider-aws   3         3         3       3            3           kubernetes.io/os=linux   2m3s
+
+NAME                                              READY   STATUS    RESTARTS   AGE
+pod/secrets-store-csi-driver-provider-aws-4jf8f   1/1     Running   0          2m2s
+pod/secrets-store-csi-driver-provider-aws-djtf5   1/1     Running   0          2m2s
+pod/secrets-store-csi-driver-provider-aws-dzg9r   1/1     Running   0          2m2s
+```
+
+You should also see that we already have created a SecretProviderClass, which is a namespaced custom resource that's used provide driver configurations and specific parameters to access your secrets stored in AWS Secrets Manger via CSI driver.
```
Comment (niallthomson) [inline website/docs/security/secrets-management/secrets-manager/ascp.md:36]: I'd suggest we get the user to create this, makes it easier for them to browse the YAML to see how it happened.

**Review by niallthomson — CHANGES_REQUESTED** [review]
Overall great content @rodrigobersa looking forward to getting this merged.

Asked for some changes for consistency and housekeeping.

File: website/docs/security/secrets-management/secrets-manager/create-secret.md (hunk 1 above)
Comment (rodrigobersa) [reply website/docs/security/secrets-management/secrets-manager/create-secret.md:8]: Agreed!

File: website/docs/security/secrets-management/secrets-manager/create-secret.md (hunk 2 above)
Comment (rodrigobersa) [reply website/docs/security/secrets-management/secrets-manager/create-secret.md:31]: Absolutely!

File: website/docs/security/secrets-management/secrets-manager/ascp.md (hunk 3 above)
Comment (rodrigobersa) [reply website/docs/security/secrets-management/secrets-manager/ascp.md:6]: 😅

**Comment by niallthomson** [comment]
Thanks @rodrigobersa !

**Pull request merged**